package.json

```diff
@@ -1,24 +1,57 @@
 {
-  "name": "shiftjs-template-template",
-  "version": "0.1.3",
-  "private": true,
+  "name": "dashboard",
+  "version": "0.1.0",
   "license": "MIT",
+  "private": true,
+  "keywords": [
+    "dashboard",
+    "chart",
+    "shiftjs",
+    "react-bootstrap",
+    "fullstack",
+    "promise",
+    "async"
+  ],
   "dependencies": {
+    "@binaris/shift-babel-macro": "0.0.5",
     "@binaris/shift-code-transform": "0.0.2",
-    "@binaris/shift-db": "^0.3.1",
+    "@binaris/shift-db": "0.5.0",
     "@binaris/shift-fetch-runtime": "0.0.4",
-    "@binaris/shift-local-proxy": "0.0.11",
-    "@binaris/shift-server-function": "0.0.1",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-scripts": "3.1.1"
+    "@binaris/shift-leveldb-server": "0.0.2",
```

Comment:
`shift-leveldb-server` is no longer required (it is imported via `shift-local-proxy`)